### DISCUSSION TITLE
Postgres - dynamically add fields to column

### DISCUSSION THREAD

**Discussion by erasebegin** [prisma/prisma#15906] [Q&A] [answered]
Hi,

I have a form that is for recording information about a person so the basics are defined in the schema (email, name, dob etc). But I want the user to be able to add new fields at will. For example maybe they want to add a "website" field for this person or (lord knows why) a "fax" field. Is this possible to do with Postgres? Is it possible to do with Prisma?

I haven't found anything like this in the documentation, and when I try to add a field that doesn't exist to a column, I get a typescript error.

Thank you 🙏

**Comment by erasebegin** [comment, marked as answer]
I think I'm going to have to just do a Json field which contains all of the additional fields

**Reply by nurul3101** [reply]
Hello @erasebegin 👋 
I agree making a field as JSON would be the most convenient and straightforward way. PrismaClient needs to know in advance all the names of the fields in the model, so this approach would be feasible.

**Reply by erasebegin** [reply]
Thanks @nurul3101, good to know that this is a reasonable way to go!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
